amundsen_application/static/js/components/TableDetail/ColumnListItem/index.tsx

```diff
@@ -73,6 +73,19 @@ export class ColumnListItem extends React.Component<
     e.stopPropagation();
   };
 
+  renderDescription = () => {
+    let expand: boolean = true;
+    if (
```

Comment:
I assume the logic you are aiming for is:
1. Return `true` if there is a description
2. Return `false` if no `description`, no `editText`, or no `editUrl`
3. Return `true` is neither apply

If so, this can be simplified as:
```
const { data, editText, editUrl } = this.props; // just for code cleanliness help
if (data.description) { // in JS this checks if a string is "truthy", which means it exists AND is not empty-string
    return true;
}
if (!editText && !editUrl) { // in JS this checks if a string is "falsy", which means it does not exist OR is empty-string
    return false;
}
return true;
```

Reply:
Should I call this logic from toggleEdit, so that toggleEdit does not work on click? Currently it toggles and size component changes.

Reply:
I got my answer, we cannot do that since a column may have stats even if it does not have description so toggle expand is required to run

Reply:
Oh I see what you mean. But here are my suggestions after taking a second look:
1. Can you instead heck for whether or not we should render something shouldn't just wrap the `EditableSection`, rather the parent <div> as well...no use rendering a parent if it won't have a child. 
2. The size changing is acceptable. We are also technically supposed to have `ColumnStats` component show up. However currently at Lyft we aren't showing column stats, and the current `ColumnStats` UI should nothing if there are no stats. So it's the `ColumnStats` rendering that would have to be improved I think if we didn't want any extra space.

Reply:
I did not understand your first point completely . But on the same lines , to stop rendering of parent if there is no child component I have put check inside toggleExpand function. Let me know if that is okay. `ColumnStats` rendering returns `null` if there are no stats.